ground-server/src/components/telemetry-adder.tsx

```diff
@@ -109,11 +109,13 @@ function StatusList({
       {
         children: <div>{status.label}</div>,
         layout: {
-          i: widgets.length.toString(),
+          i: Date.now().toString(),
```

Comment:
Is this ordering by time added, or am I misinterpreting that?

Reply:
It's not ordering, each widget just needs a unique id that it can be referred to. Before I just did the length of the array, which worked fine until I added deleting. When I added deleting, then it is easy to create two widgets with same ID. So, now I just use the milliseconds-since-epoch at which it was created as the id